app/assets/stylesheets/styles/post-edit.scss

```diff
@@ -10,16 +10,16 @@ label {
   margin: 0;
 }
 
-#container {
+.editContainer {
   display: grid;
   grid-template-columns: 12.5% 75% 12.5%;
 }
 
-#header-container {
+.editHeaderContainer {
   margin-top: 10px;
```

Comment:
kinda goes for the whole file but you want to avoid non-relative unit stying

Reply:
Why? We're targeting this to be used on full resolution desktops to be used by the eboard and members we trust to submit posts. This is not going to be a mobile friendly app especially with the drag and drop image behavior. Anyway with this PR merged please take the non-relative unit styling discussion to Issue #21 so we can maybe include this once we make styling improvements to the post editor.